handlers/main.yml

```diff
@@ -17,6 +17,13 @@
     state: started
     enabled: True
 
+- name: Start celery worker
+  systemd:
+    daemon_reload: yes
+    name: "{{ django_stack_celery_svc_name }}.service"
+    state: restarted
+    enabled: True
```

Comment:
Handler should be called `restart celery worker`, since that's what it does. :ok_hand: on the `daemon_reload` option!